test/parallel/test-trace-events-worker-metadata.js

```diff
@@ -23,7 +23,7 @@ if (isMainThread) {
       assert(traces.length > 0);
       assert(traces.some((trace) =>
         trace.cat === '__metadata' && trace.name === 'thread_name' &&
-          trace.args.name === '[worker 1]'));
```

Comment:
There doesn't appear to be a test of the new `name` option when creating a Worker Thread? Or did I just miss it?

Reply:
There's already a `name` option when creating a Worker Thread (https://nodejs.org/api/worker_threads.html#new-workerfilename-options), but its usage was only to give a name to the worker on tracing logs. I have set a default value for this option to `WorkerThread` so I can also call `uv_thread_setname()` with this value and users could specific the value they want to it.